src/components/MoodForm/MoodForm.tsx

```diff
@@ -1,51 +1,54 @@
-import './MoodForm.css';
-import { useState } from 'react';
-import { useMutation } from '@apollo/client';
-import { SUBMIT_MOOD } from '../../utils/graph_mutations';
+import "./MoodForm.css"
+import { useState, useContext } from "react"
+import { useMutation } from "@apollo/client"
+import { SUBMIT_MOOD } from "../../utils/graph_mutations"
+import { AppContext } from "../../utils/context"
 
 const MoodForm = () => {
-  const [ mood, setMood ] = useState('')
-  const [ description, setDescription ] = useState('')
-  const [ validateForm, setValidateForm ] = useState(true)
-  const [ createMood ] = useMutation(SUBMIT_MOOD)
-  const [isSubmitted, setIsSubmitted ] = useState(false)
+  const [mood, setMood] = useState("")
+  const [description, setDescription] = useState("")
+  const [validateForm, setValidateForm] = useState(true)
+  const [createMood] = useMutation(SUBMIT_MOOD)
+  // const [isSubmitted, setIsSubmitted] = useState(false)
+  const { todaysMood, displayMood } = useContext(AppContext)
 
   const handleSubmit = (e: React.FormEvent<HTMLFormElement>) => {
-    e.preventDefault();
+    e.preventDefault()
 
     if (!mood) {
       setValidateForm(false)
     } else {
-      createMood({ variables: { mood: parseInt(mood), description: description }})
+      createMood({ variables: { mood: parseInt(mood), description: description } })
 
       setValidateForm(true)
-      setMood('');
-      setDescription('');
-      setIsSubmitted(true)
+      setMood("")
+      setDescription("")
+      // setIsSubmitted(true)
     }
   }
 
   return (
     <section className="mood-form-container">
-      { !isSubmitted ?
-        <form className="mood-form" onSubmit={ handleSubmit }>
-          <h2>
-            How are you feeling today?
-          </h2>
+      {todaysMood ? (
+        <div>
+          <p>I am feeling: {displayMood(todaysMood!.mood)}</p>
```

Comment:
We can make this piece a component since we reuse this in both MoodForm and Dashboard.

Reply:
This would be a good change to make

Reply:
New component added